tests/Unit/Domain/Python/CMakeLists.txt

```diff
@@ -31,6 +31,12 @@ spectre_add_python_bindings_test(
   "Unit;Domain"
   PyDomain)
 
+spectre_add_python_bindings_test(
+  "Unit.Domain.Python.JacobianDiagnostic"
+  Test_JacobianDiagnosticPythonBinding.py
```

Comment:
Please just name this Test_JacobianDiagnostic.py and move the other file up one directory level. Also, name the other file just `JacobianDiagnostic.py`. That's what we do for all other random-value tests.